package.json

```diff
@@ -18,15 +18,17 @@
     "type-check": "yarn workspaces foreach --all --parallel run type-check"
   },
   "dependencies": {
-    "@shapeshiftoss/caip": "workspace:^",
+    "@shapeshiftoss/caip": "^8.16.0",
     "@shapeshiftoss/types": "workspace:^",
     "@shapeshiftoss/utils": "workspace:^",
     "bignumber.js": "^9.3.0",
+    "lodash": "^4.17.21",
```

Comment:
@shapeshifoss/caip wasn't published with the necessary dependencies in place (monorepo hoisted dependencies setup), so adding here for now. We may use lodash at some point, or we can always re-publish caip and remove accordingly.